tests/test_flask.py

```diff
@@ -5,6 +5,7 @@
 from vitality.database import Database, password_sha256
 from vitality.trainee import Trainee
 from vitality.trainer import Trainer
+from vitality.database import InvalidCharactersException
```

Comment:
Append InvalidCharactersException to the end of line 5 since we already have a `from .. import...` for database.